src/test/lib/pgSQL-tests.js

```diff
@@ -0,0 +1,425 @@
+/**************************************************************
+ *
+ * UNIT TESTS postgresQL
+ *
+ **************************************************************/
+
+// external modules
+const assert = require('assert');
+const config = require('../../config/config');
+const async = require('async');
+pg=require('../../lib/postgresQL')(config.pg);
+var createTable=[
+    `CREATE TABLE IF NOT EXISTS repositories (id serial PRIMARY KEY, name varchar NOT NULL,
+        domain varchar NOT NULL, contact varchar NOT NULL, token varchar NOT NULL);`,
+    'CREATE INDEX IF NOT EXISTS repositories_name_idx ON repositories(name);',
+    'CREATE INDEX IF NOT EXISTS repositories_domain_idx ON repositories(domain);',
+    'CREATE INDEX IF NOT EXISTS repositories_contact_idx ON repositories(contact);',
+    'CREATE INDEX IF NOT EXISTS repositories_token_idx ON repositories(token);'];
```

Comment:
Closing bracket put in new line
```suggestion
    'CREATE INDEX IF NOT EXISTS repositories_token_idx ON repositories(token);'
];
```